backend/main.py

```diff
@@ -1,6 +1,7 @@
 from fastapi import FastAPI
+from backend.routers.user import router  # import the user router
```

Comment:
In backend/routers/__init__.py: put in ``from .user import router as UserRouter``
Thecn change this import ``from backend.routers import UserRouter``